### ISSUE TITLE
Version 6.8 has a PHP requirement incompatible with your PHP version, PHP extensions and Composer version 

### ISSUE THREAD

**Issue by David-Baron** [deployphp/deployer#2480]
- Deployer version: 6.8
- Deployment OS: windows
- php 8.0.3
- symfony 5.2
- composer v2

When: composer require deployer/deployer ^6

Console message: 
Package deployer/deployer at version 6.8 has a PHP requirement incompatible with your PHP version, PHP extensions and Composer version

**Comment by David-Baron** [comment]
Voici le trace:

```shell
PS C:\Users\SoS Consoles\Desktop\ouebsite> composer require deployer/deployer ^6.8 -vvv
Reading ./composer.json (C:\Users\SoS Consoles\Desktop\ouebsite\composer.json)
Loading config file C:/Users/SoS Consoles/AppData/Roaming/Composer/config.json
Loading config file ./composer.json (C:\Users\SoS Consoles\Desktop\ouebsite\composer.json)
Checked CA file C:\Users\SoS Consoles\AppData\Local\Temp\opeB9A6.tmp: valid
Executing command (C:\Users\SoS Consoles\Desktop\ouebsite): git branch -a --no-color --no-abbrev -v
Reading C:/Users/SoS Consoles/AppData/Roaming/Composer/composer.json (C:\Users\SoS Consoles\AppData\Roaming\Composer\composer.json)
Loading config file C:/Users/SoS Consoles/AppData/Roaming/Composer/config.json
Loading config file C:/Users/SoS Consoles/AppData/Roaming/Composer/composer.json (C:\Users\SoS Consoles\AppData\Roaming\Composer\composer.json)
Reading C:\Users\SoS Consoles\Desktop\ouebsite/vendor/composer/installed.json (C:\Users\SoS Consoles\Desktop\ouebsite\vendor\composer\installed.json)
Reading C:/Users/SoS Consoles/AppData/Roaming/Composer/vendor/composer/installed.json (C:\Users\SoS Consoles\AppData\Roaming\Composer\vendor\composer\installed.json)
Loading plugin PackageVersions\Installer
Loading plugin Symfony\Flex\Flex
Running 2.0.10 (2021-02-23 16:11:37) with PHP 8.0.3 on Windows NT / 10.0
Reading C:/Users/SoS Consoles/AppData/Local/Composer/repo/https---flex.symfony.com/aliases.json from cache
Downloading https://flex.symfony.com/aliases.json if modified
[200] https://flex.symfony.com/aliases.json
Writing C:/Users/SoS Consoles/AppData/Local/Composer/repo/https---flex.symfony.com/aliases.json into cache
Downloading https://repo.packagist.org/packages.json
[200] https://repo.packagist.org/packages.json
Writing C:/Users/SoS Consoles/AppData/Local/Composer/repo/https---repo.packagist.org/packages.json into cache
Reading C:/Users/SoS Consoles/AppData/Local/Composer/repo/https---repo.packagist.org/provider-deployer~deployer.json from cache
Downloading https://repo.packagist.org/p2/deployer/deployer.json if modified
Reading C:/Users/SoS Consoles/AppData/Local/Composer/repo/https---repo.packagist.org/provider-deployer~deployer~dev.json from cache
Downloading https://repo.packagist.org/p2/deployer/deployer~dev.json if modified
[304] https://repo.packagist.org/p2/deployer/deployer.json
[304] https://repo.packagist.org/p2/deployer/deployer~dev.json
Reading C:/Users/SoS Consoles/AppData/Local/Composer/repo/https---repo.packagist.org/provider-deployer~deployer.json from cache
Reading C:/Users/SoS Consoles/AppData/Local/Composer/repo/https---repo.packagist.org/provider-deployer~deployer~dev.json from cache


  [InvalidArgumentException]
  Package deployer/deployer at version 6.8 has a PHP requirement incompatible with your PHP version, PHP extensions and Composer version  


Exception trace:
 () at phar://C:/ProgramData/ComposerSetup/bin/composer.phar/src/Composer/Command/InitCommand.php:749
 Composer\Command\InitCommand->findBestVersionAndNameForPackage() at phar://C:/ProgramData/ComposerSetup/bin/composer.phar/src/Composer/Command/InitCommand.php:443
 Composer\Command\InitCommand->determineRequirements() at phar://C:/ProgramData/ComposerSetup/bin/composer.phar/src/Composer/Command/RequireCommand.php:189
 Composer\Command\RequireCommand->execute() at C:\Users\SoS Consoles\Desktop\ouebsite\vendor\symfony\flex\src\Command\RequireCommand.php:54
 Symfony\Flex\Command\RequireCommand->execute() at phar://C:/ProgramData/ComposerSetup/bin/composer.phar/vendor/symfony/console/Command/Command.php:245
 Symfony\Component\Console\Command\Command->run() at phar://C:/ProgramData/ComposerSetup/bin/composer.phar/vendor/symfony/console/Application.php:835
 Symfony\Component\Console\Application->doRunCommand() at phar://C:/ProgramData/ComposerSetup/bin/composer.phar/vendor/symfony/console/Application.php:185
 Symfony\Component\Console\Application->doRun() at phar://C:/ProgramData/ComposerSetup/bin/composer.phar/src/Composer/Console/Application.php:310
 Composer\Console\Application->doRun() at phar://C:/ProgramData/ComposerSetup/bin/composer.phar/vendor/symfony/console/Application.php:117
 Symfony\Component\Console\Application->run() at phar://C:/ProgramData/ComposerSetup/bin/composer.phar/src/Composer/Console/Application.php:122
 Composer\Console\Application->run() at phar://C:/ProgramData/ComposerSetup/bin/composer.phar/bin/composer:63
 require() at C:\ProgramData\ComposerSetup\bin\composer.phar:24
```

**Comment by David-Baron** [comment]
Resolved by require "deployer/deployer": "^7.0",
